cpp/custrings/text/edit_distance.cu

```diff
@@ -116,6 +120,32 @@ struct editdistance_levenshtein_algorithm
     }
 };
 
+struct editdistance_matrix_levenshtein_algorithm : editdistance_levenshtein_algorithm {
+
+    unsigned int count_;   // size of nvstrings object
```

Comment:
No need for the trailing underscore (even in the constructor initialization list `count(count)` is legal).

```suggestion
    unsigned int count;   // size of nvstrings object
```